spec/view_objects/allocations_view_spec.rb

```diff
@@ -0,0 +1,368 @@
+require "rails_helper"
+
+describe AllocationsView do
+  let(:training_provider) { build(:provider, provider_name: "Training Provider 1") }
+  let(:another_training_provider) { build(:provider, provider_name: "Training Provider 2") }
+  let(:accredited_body) { build(:provider) }
+  let(:training_providers) { [training_provider, another_training_provider] }
+
+  describe "#allocation_renewals" do
+    subject { AllocationsView.new(training_providers: training_providers, allocations: allocations).repeat_allocation_statuses }
+
+    context "Accrediting provider has re-requested an allocation for a training provider" do
+      let(:repeat_allocation) do
+        build(:allocation, :repeat, accredited_body: accredited_body, provider: training_provider, number_of_places: 1)
+      end
+      let(:initial_allocation) do
+        build(:allocation, :initial, accredited_body: accredited_body, provider: another_training_provider, number_of_places: 3)
+      end
+      let(:allocations) { [repeat_allocation, initial_allocation] }
+
+      it {
```

Comment:
I think all the multiline `it` scenarios should use `do end` blocks

Reply:
done, through we should add a linter rule for that next time